sample-config.json

```diff
@@ -27,5 +27,6 @@
     "append_date_to_prefix_grain": "day",
     "append_date_to_filename": true,
     "append_date_to_filename_grain": "microsecond",
-    "flatten_records": false
+    "flattening_enabled": false,
```

Comment:
@prakharcode are these properties automatically read in to the SDK? I noticed they aren't configured within the target itself `target.py`

Reply:
yes: https://github.com/search?q=repo%3Ameltano%2Fsdk%20get_flattening_options&type=code